features/tab-access-tabs.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Adjust tab stops
+  In order to customize tab stops
+  As a developer using python-docx
+  I need ways to create, read, update, and delete tab stops 
+
+
+  @wip
+  Scenario Outline: Count tab stop objects
+    Given a tab_stops having <count> tab stops
+      Then len(tab_stops) is <count>
+      
+    Examples:  values for tab_stop elements
+      | count |
+      | 0     |
+      | 3     |
+      
+
+  @wip
+  Scenario: Access individual tab stops in tab_stops
+    Given a tab_stops having 3 tab stops
+     Then I can iterate over the tab_stop elements
+      And I can access a tab_stop by index
+
+
+  @wip
+  Scenario Outline: Add a tab stop
+    Given a tab_stops having <count> tab stops
+     When I add a tab_stop
+     Then len(tab_stops) is <lenval>
```

Comment:
I think on this one we need another "Then" step (actually would start with "And") to test that they're ordered by increasing distance, don't we?